### ISSUE TITLE
Perhaps record landslides vs skill difference

### ISSUE THREAD

**Issue by kozzy1981** [Baldrickk/replay_team_balance#8]
Would be interesting to see how often a game with a 12 tank (or whatever) score difference occurs and what the 'skill' difference is in these 'landslides' to see where the correlations lie between the two.

**Comment by Baldrickk** [comment]
Interesting idea.
It could plot result  (as per kill differences) against team rating difference

**Comment by Baldrickk** [comment]
Right, I think that this should do the job.  What do you think @kozzy1981 ?
